index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/7967e81f77.js" crossorigin="anonymous"></script>
+    <title>Portfolio</title>
+</head>
+<body>
+   <main class="container">
+       <header>
+           <nav class="navItems">
+            <a class="logo" href="">Welcome</a>
```

Comment:
- [OPTIONAL] kindly add the font-style of your logo so that it looks like the Figma model